app.rb

```diff
@@ -0,0 +1,179 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @num = 0
+  end
+
+  def start
+    puts 'Welcome to School Library!'
+    menu while @num != 7
+  end
+
+  private
+
+  def menu
+    until @num < 7 && @num.positive?
+      puts "Please choose an option by entering a number\n1- List all books\n2- List all people\n3- Create a person
+4- Create a book\n5- Create a rental\n6- List all rentals\n7- exit"
+      @num = gets.chomp.to_i
+    end
```

Comment:
- [ ] [OPTIONAL] Because the previous CR didn't mention this it is not a requirement however I do strongly suggest you fix this before moving on to the next project. the user is not able to exit the application due to this `until` loop, please delete the loop in order to fix this, to be more specific you only need to delete lines 22 and 26 the code inside the loop should remain there.

Reply:
- [ ] [OPTONAL] App exits fine now, great implementation 👍🏽  However, we can make it more user-friendly by adding some goodbye message to display to the user.